WzComparerR2/CharaSimControl/ItemTooltipRender2.cs

```diff
@@ -281,7 +281,13 @@ private Bitmap RenderItem(out int picH)
             if (!string.IsNullOrEmpty(attr))
             {
                 g.DrawString(attr, GearGraphics.ItemDetailFont, GearGraphics.GearNameBrushC, tooltip.Width / 2, picH, format);
-                picH += 19;
+                if (attr.Contains("\n")) {
+                    picH += 31;
```

Comment:
Just curious why the line height is not 16, do you have carefully measured it? 👀

Reply:
Line spacing from g.DrawString is smaller than in game result, and that's why picH is not multiples of 16. Refactor of ItemTooltipRender2 and GearTooltipRender2 may required.

Reply:
OK I see. It is hard to draw string with custom line spacing so just ignore it.